hack/build.sh

```diff
@@ -16,6 +16,9 @@
 
 set -o pipefail
 
+[[ ! -v REPO_ROOT_DIR ]] && REPO_ROOT_DIR="$(git rev-parse --show-toplevel)"
+readonly REPO_ROOT_DIR
```

Comment:
That won't work on macOS. You would need to backport my fix, too then.

Reply:
fixed